app/tsconfig.json

```diff
@@ -1,21 +1,24 @@
 {
   "compilerOptions": {
-      "allowJs": true,
-      "declaration": false,
-      "esModuleInterop": true,
-      "incremental": true,
-      "module": "commonjs",
-      "moduleResolution": "node",
-      "outDir": "./dist",
-      "resolveJsonModule": true,
-      "skipLibCheck": true,
-      "sourceMap": true,
-      // https://stackoverflow.com/questions/51716406/typescript-tsconfig-settings-for-node-js-10
-      // and 'dom' to tell tsc it's okay to use the URL object (which is in Node >= 7)
-      "target": "es2018",
-      "lib": ["es2018", "dom"]
+    "allowJs": true,
+    "declaration": false,
+    "esModuleInterop": true,
+    "incremental": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "outDir": "./dist",
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "sourceMap": true,
+    // https://stackoverflow.com/questions/51716406/typescript-tsconfig-settings-for-node-js-10
+    // and 'dom' to tell tsc it's okay to use the URL object (which is in Node >= 7)
+    "target": "es2019",
+    "lib": [
+      "es2020",
+      "dom"
+    ]
```

Comment:
Careful with this, this will break users that for some reason keep using old versions of Electron (which come with older Node/v8).

Reply:
> Careful with this, this will break users that for some reason keep using old versions of Electron (which come with older Node/v8).

Ahh, was just trying to align it with the one is the actual project. Do we want to support that use case? If so, I can revert.

Reply:
> Ahh, was just trying to align it with the one is the actual project. Do we want to support that use case? If so, I can revert.

@TheCleric got it. This tsconfig.json should (like the main one) be bump as rarely as possible, to support running on old runtimes. Here in `app`, "runtime" means "the chosen Electron", whereas in `.`, "runtime" means "the node.js version of the user"

So, there's no alignment to aim at necessarily, we can/should bump it if an `app` dep needs it, but before this happens it's actually preferable to let it be, for users to keep being able to use old Electrons.

Adding a comment to clarify this.